docs/en/latest/tutorials/monitor-api-health-check.md

```diff
@@ -0,0 +1,192 @@
+---
+title: Monitor API Health Check with Prometheus
+keywords:
+  - API Health Check
+  - Monitoring with Prometheus
+  - API Gateway
+description: In this tutorial, we'll guide you on how to enable and monitor API health checks using APISIX and Prometheus.
+---
+
+<!--
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+-->
+
+[APISIX](https://apisix.apache.org/) has a [health check](https://apisix.apache.org/docs/apisix/tutorials/health-check/) mechanism, which proactively checks the health status of the upstream nodes in your system. Also, APISIX integrates with [Prometheus](https://prometheus.io/) through its [plugin](https://apisix.apache.org/docs/apisix/plugins/prometheus/) that exposes upstream nodes (multiple instances of a backend API service that APISIX manages) health check metrics on the Prometheus metrics endpoint typically, on URL path **`/apisix/prometheus/metrics`**.
+
+In this tutorial, we'll guide you on how to **enable and monitor API health checks** using APISIX and Prometheus.
+
+## Prerequisite(s)
+
+- Before you start, it is good to have a basic understanding of APISIX. Familiarity with [API gateway](https://apisix.apache.org/docs/apisix/terminology/api-gateway/), and its key concepts such as [routes](https://docs.api7.ai/apisix/key-concepts/routes), [upstream](https://docs.api7.ai/apisix/key-concepts/upstreams), [Admin API](https://apisix.apache.org/docs/apisix/admin-api/), [plugins](https://docs.api7.ai/apisix/key-concepts/plugins), and HTTP protocol will also be beneficial.
+- [Docker](https://docs.docker.com/get-docker/) is used to install the containerized etcd and APISIX.
+- Install [cURL](https://curl.se/) to send requests to the services for validation.
+
+## Start the APISIX demo project
+
+This project leverages existing the pre-defined [Docker Compose configuration](https://github.com/apache/apisix-docker/blob/master/example/docker-compose.yml) file to set up, deploy and run APISIX, etcd, Prometheus, and other services with a single command. First, clone the [apisix-docker](https://github.com/apache/apisix-docker) repo on GitHub and open it in your favorite editor, navigate to `/example` folder, and start the project by simply running `docker compose up` from the folder.
```

Comment:
>This project leverages existing the pre-defined [Docker Compose configuration]

This project leverages ~existing~ the pre-defined [Docker Compose configuration

Reply:
Grammarly flags this for me but not for you 🤔

Reply:
Honestly, it does not flag. Maybe because I use the free version:) Also, I assume that there are no critical grammar issues in this tutorial.

Reply:
This one I removed as your Grammarly suggest.